avocado/utils/distro.py

```diff
@@ -442,7 +442,7 @@ class UnionTechProbe(Probe):
     Simple probe for UnionTech systems in general
     """
 
-    CHECK_FILE = '/etc/UnionTech-release'
+    CHECK_FILE = '/etc/os-version'
```

Comment:
There is a typo on the commit message. Can you also please briefly describe this change is about the UnionTech distro probe there?

Reply:
There are multiple distributions of UOS, and there are multiple branches. "/etc/uniontech-release" is not universal, so it needs to be changed

Reply:
@beraldoleal

Reply:
@beraldoleal ,commit message has been modified

Reply:
@shilei522 Hi! The typo is still there. The explanation for the change is also missing. Could you please expand your what you're trying to do?

Reply:
@shilei522 the commit message has "updta the CHECK_FILE" and without a brief description as pointed by @ana.

Please consider reading our [Commit Style Guide](https://avocado-framework.readthedocs.io/en/94.0/guides/contributor/chapters/styleguides.html#commit-style-guide).

Reply:
@ana @beraldoleal ,Look again, please. Is that right?